schema.graphql

```diff
@@ -87,6 +99,21 @@ type CourseSection {
   startDate: Date!
 }
 
+input CreateTeachingPreferenceInput {
+  CoursePreferenceInput: [CoursePreferenceInput!]!
```

Comment:
this should be named `courses` according to company 3 shared schema

Reply:
Corrected, thanks!